tests/test_wave.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
```

Comment:
There are some general tests for FileType subclasses: https://github.com/quodlibet/mutagen/blob/8fe2946fb048b22253013c5833eef42823308278/tests/test___init__.py#L468

Try adding the WAVE there with the test files.